internal/middleware/connlimit.go

```diff
@@ -45,3 +48,11 @@ func ConnLimit(node *centrifuge.Node, ruleContainer *rule.Container, h http.Hand
 		h.ServeHTTP(w, r)
 	})
 }
+
+func rateLimiter(connRateLimit int) ratelimit.Limiter {
+	if connRateLimit > 0 {
+		// Configuring the limiter with slack to perform delays only under high load
+		return ratelimit.New(connRateLimit, ratelimit.WithSlack(connRateLimit))
```

Comment:
Should we use `ratelimit.Per(time.Second)` here (or nothing since per second is default), or there is some logic behind using `WithSlack `?

Reply:
`Per(time.Second)` is by default anyway. `WithSlack` allows the limiter to accumulate "unspent" requests for future traffic bursts. Let's assume, we create a limiter allowing 100 requests per second. It means that we will always wait for 10ms between each request. But usually, the traffic is unevenly distributed. Using`WithSlack` we accumulate "unspent" requests from the 1st second and can spend them during the 2nd second without delays. 
Since we commonly have a limit higher than the usual load, we always will have some "buffer" to process requests without delay.